app/views/course/user_invitations/_invitation_fields.html.slim

```diff
@@ -1,4 +1,10 @@
 = content_tag_for(:tr, f.object, class: 'nested-fields') do
   td = f.input :name, label: false, input_html: { class: 'invitation_name' }
   td = f.input :email, label: false, input_html: { class: 'invitation_email' }
+  td = f.input :role,
+               as: :select,
+               collection: Course::UserInvitation.roles.keys,
```

Comment:
Can use translations here instead of the keys. I think there are some defined under `course/users.yml` in the `config/locales...` folder.

Reply:
I looked through them, but it doesn't seem to exist.... Not sure if I'm missing anything here. The closest I've seen was in `app/views/course/users/staff.html.slim`, which takes a constant defined in the model for line 15, and `keys` for line 37.

Reply:
`config/locales/en/course/users.yml` - to invoke, you prob need to specify the full key, like `Course::UserInvitation.roles.map { |role| t("course.users.role.#{role}") }`

Reply:
Ah ok, missed that one.

The form's field value will still have to be either the key or the value from `Course::UserInvitation.roles`. So what I'll do is to use `label_method` to change the display with translations, while keeping the keys as values.

Reply:
If I remember correctly, you can pass the key and the translation in arrays `[['student', 'Translation of Student'], [ 'staff', 'Translation of Staff']]` to the collection.

Reply:
Hmm I think to do that wouldn't be too different from what I have now. `label_method` was supposed to map values to label names in the first place.

Reply:
👍 Yep, that's fine